internal/pkg/cli/deploy.go

```diff
@@ -789,7 +793,7 @@ func BuildDeployCmd() *cobra.Command {
 
 	cmd.Flags().BoolVar(&deployEnvironment, deployEnvFlag, false, deployEnvFlagDescription)
 	cmd.Flags().BoolVar(&initEnvironment, yesInitEnvFlag, false, yesInitEnvFlagDescription)
-	cmd.Flags().BoolVar(&initWorkload, yesInitWorkloadFlag, false, yesInitWorkloadFlagDescription)
+	cmd.Flags().BoolVar(&vars.yesInitWkld, yesInitWorkloadFlag, false, yesInitWorkloadFlagDescription)
```

Comment:
Is the default `true`?

Reply:
no, I think we should err on the side of only deploying initted local workloads unless the customer opts in.